base/Makefile

```diff
@@ -130,6 +130,14 @@ resolve_path += && \
 	$1_wd=`dirname "$${$1}"`; $1=$${$1_}; \
 	if [ -z "`echo $${$1} | grep '^/'`" ]; then $1=$${$1_wd}/$${$1}; fi; \
 	fi
+else
+# try to use the SO_NAME (if the named file exists)
+resolve_path += && \
+	$1_=`objdump -p "$${$1}" | awk '/SONAME/ {print $$2}'` && \
```

Comment:
I guess Awk should be added to the README section on required build tools?

Reply:
I actually wonder if `awk` is not required by POSIX?  It's builtin to `busybox`, which is my poor-mans litmus test to know whether or not it's available everywhere.  ;)

Reply:
fwiw, we already use it (to generate julia_version.h) – I checked before I used it here :)